mainchain-follower/db-sync-follower/src/db_model.rs

```diff
@@ -260,6 +272,28 @@ impl<'r> Decode<'r, Postgres> for StakeDelegation {
 	}
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub(crate) struct NativeTokenAmount(pub u64);
+impl From<NativeTokenAmount> for sidechain_domain::NativeTokenAmount {
+	fn from(value: NativeTokenAmount) -> Self {
+		Self(value.0)
+	}
+}
+
+impl sqlx::Type<Postgres> for NativeTokenAmount {
+	fn type_info() -> <Postgres as sqlx::Database>::TypeInfo {
+		PgTypeInfo::with_name("NUMERIC")
+	}
+}
+
+impl<'r> Decode<'r, Postgres> for NativeTokenAmount {
+	fn decode(value: <Postgres as HasValueRef<'r>>::ValueRef) -> Result<Self, BoxDynError> {
+		let decoded = <sqlx::types::BigDecimal as Decode<Postgres>>::decode(value)?;
+		let i = decoded.to_u64().ok_or("NativeTokenQuantity is always a u64".to_string())?;
```

Comment:
It is not asset of policy a4db1b5937a71ee33934f2f1371c9ef54fdedcf2cfab44a17eddcfab and name 54454459745f54656464795f4c71 has sum of 1400891858530927120255690

One transaction cannot in fact exceed 2**64 but you can have a lot of transactions summed.

User query:
```
SELECT
	multi_asset.policy,
	multi_asset.name,
    SUM(ma_tx_out.quantity) as s
FROM tx_out
LEFT JOIN ma_tx_out    ON ma_tx_out.tx_out_id = tx_out.id
LEFT JOIN multi_asset  ON multi_asset.id = ma_tx_out.ident
GROUP BY multi_asset.policy, multi_asset.name
ORDER BY s DESC;
```

Reply:
Changed it to u128